lib/std/zig/WindowsSdk.zig

```diff
@@ -976,14 +975,14 @@ const MsvcLibDir = struct {
         };
         errdefer base_path.deinit();
 
-        const folder_with_arch = "\\VC\\lib\\" ++ comptime switch (builtin.target.cpu.arch) {
+        try base_path.appendSlice("\\VC\\lib\\");
+        try base_path.appendSlice(switch (arch) {
             .thumb => "arm",
             .aarch64 => "arm64",
-            .x86 => "", //x86 is in the root of the Lib folder
+            .x86 => "x86",
```

Comment:
I changed this because I couldn't find any evidence that this was true on my system at least. Perhaps someone more familiar with this can clarify?

Reply:
Old versions were different and had a different structure, newer versions such as the one on your system use the `findViaRegistry` code path instead.

Reply:
Ok, I'll revert this line then.